src/Client.ts

```diff
@@ -248,7 +248,8 @@ export default class Client {
     peerAddress: string,
     opts?: Partial<NetworkOptions>
   ): Promise<boolean> {
-    const apiUrl = opts?.apiUrl || ApiUrls[opts?.env || 'dev']
+    const apiUrl =
+      opts?.apiUrl || process.env.XMTP_API_URL || ApiUrls[opts?.env || 'dev']
```

Comment:
Out of curiosity: where is `process.env.XMTP_API_URL` set?

Reply:
I was passing it in manually with `XMTP_API_URL=http://localhost:5555 npm test`. It doesn't really need to be here though, I was just testing something locally with it. I can either remove it or at least add a note to the README about using it if we want to keep it